src/Microsoft.ML.KMeansClustering/KMeansModelParameters.cs

```diff
@@ -13,13 +13,13 @@
 using Microsoft.ML.Model;
 using Microsoft.ML.Model.OnnxConverter;
 using Microsoft.ML.Numeric;
-using Microsoft.ML.Trainers.KMeans;
+using Microsoft.ML.Trainers;
 using Float = System.Single;
 
```

Comment:
Ugh. One of you and @jwood803 are going to make each other very unhappy with the rebase conflicts. :D

Reply:
I don't mind doing the merge on my pull request. :)